src/runtime_src/core/tools/common/ReportMemory.cpp

```diff
@@ -225,6 +244,20 @@ populate_memtopology(const xrt_core::device * device, const std::string& desc)
   }
   pt.add_child(std::string("board.memory.data_streams"), ptStream_array);
   pt.add_child(std::string("board.memory.memories"), ptMem_array );
+   
+  // Populate softkernel mem stats 
+  boost::property_tree::ptree ptSkMem;
+  std::stringstream ss(mm_buf[map->m_count + 1]);
+  if (!ss.str().empty()) {
+    uint64_t hboCnt, mapboCnt, unmapboCnt, freeboCnt;
+  
+    ss >> hboCnt >> mapboCnt >> unmapboCnt >> freeboCnt;
+    ptSkMem.put("sk_hbo", hboCnt);
+    ptSkMem.put("sk_mapbo", mapboCnt);
+    ptSkMem.put("sk_unmapbo", unmapboCnt);
+    ptSkMem.put("sk_freebo", freeboCnt);
+    pt.add_child(std::string("board.memory.sk_memories"), ptSkMem);

```

Comment:
sk_memory_usage would be better instead of sk_memories

Reply:
Updated